src/Bicep.LangServer/Completions/SyntaxMatcher.cs

```diff
@@ -174,16 +174,17 @@ public static List<SyntaxBase> FindNodesMatchingOffset(SyntaxBase syntax, int of
             return nodes;
         }
 
-        public static ImmutableArray<SyntaxBase> FindNodesInRange(ProgramSyntax syntax, int startOffset, int endOffset)
+        // Finds syntax nodes that encompass the entire range (i.e. are found at both the start and end of
+        //   the range)
+        public static List<SyntaxBase> FindNodesSpanningRange(ProgramSyntax syntax, int startOffset, int endOffset)
```

Comment:
I'm curious, did this get changed to List due to perf concerns?

Reply:
Because FindLastNodeOfType() in the same file is incompatible with its arguments.